openscenario/openscenario_interpreter/src/openscenario_interpreter.cpp

```diff
@@ -39,13 +39,15 @@ Interpreter::Interpreter(const rclcpp::NodeOptions & options)
   local_real_time_factor(1.0),
   osc_path(""),
   output_directory("/tmp"),
-  record(false)
+  record(false),
+  publish_empty_context(false)
 {
   DECLARE_PARAMETER(local_frame_rate);
   DECLARE_PARAMETER(local_real_time_factor);
   DECLARE_PARAMETER(osc_path);
   DECLARE_PARAMETER(output_directory);
   DECLARE_PARAMETER(record);
+  DECLARE_PARAMETER(publish_empty_context);
```

Comment:
Unless you have a technical reason not to, rearrange this line so that it is in lexicographical order with the other DECLARE_PARAMETERs, and also change the declaration order of class data members to be the same as this DECLARE_PARAMETER.

Reply:
Fixed in https://github.com/tier4/scenario_simulator_v2/pull/1305/commits/a5a5f8854f3e9e549c8f0594fa0e8dffcfc8225d

Reply:
Fixed in https://github.com/tier4/scenario_simulator_v2/pull/1305/commits/6bf5aebe268b71d0fef3e35cedd878abbf3b8ac0